### PR TITLE
TTS Always On feature proposed from Pythonesa's issue 

### PR DESCRIPTION
**Pull request by chrisvdev**
Added new global variable that enable/disable the TTS Always On Feature and the current logic for that

### CODE REVIEW

**Review by felixicaza — COMMENTED** [review]
Genial 🚀

**Review by Pythonesa — APPROVED** [review]
LG2me

**Review by iSMP14 — COMMENTED** [review]
Nice code ♥

**Review by Pythonesa — APPROVED** [review]
LG2me

**Review by Pythonesa — APPROVED** [review]
oki doki

File: src/commands/speak.js [hunk 1]
```diff
@@ -1,92 +1,102 @@
...
+// eslint-disable-next-line no-unused-vars
+const [ACCENT_OR_MODIFIER, VARIANT] = [0, 1]
 
-  setConfig(userName, accent, variant) {
-    if (tts.isAValidVoice(accent)) {
-      this.#vault[userName] = { accent, variant }
-      localStorage.setItem(VAULT, JSON.stringify(this.#vault))
-    }
-  }
+const isAAccent = /^[a-zA-Z]{1,2}-[a-zA-Z]{1,2}$/g
```
Comment (JavierBalonga) [inline src/commands/speak.js:14]: isAAccent? 👀 no deberia ser `isAccent`

**Pull request merged**